platform/android/java_godot_lib_jni.cpp

```diff
@@ -201,7 +201,7 @@ jvalret _variant_to_jvalue(JNIEnv *env, Variant::Type p_type, const Variant *p_a
 			v.obj = arr;
 
 		} break;
-		case Variant::PACKED_REAL_ARRAY: {
+		case Variant::PACKED_FLOAT32_ARRAY: {
```

Comment:
Missing 64-bit.

Reply:
I will add a warning because I have no idea how to do this